app/assets/stylesheets/common/base/sidebar.scss

```diff
@@ -31,13 +31,19 @@
 
   @include unselectable;
 
+  // 100dvh with fallback for old browsers
+  --100dvh: 100vh;
+  @supports (height: 100dvh) {
+    --100dvh: 100dvh;
+  }
+
   .footer-nav-ipad & {
     top: calc(var(--header-offset) + var(--footer-nav-height));
     height: calc(
-      100vh - var(--header-offset, 0px) - var(--footer-nav-height, 0px)
+      var(--100dvh) - var(--header-offset, 0px) - var(--footer-nav-height, 0px)
     );
   }
-  height: calc(100vh - var(--header-offset, 0px));
+  height: calc(var(--100dvh) - var(--header-offset, 0px));
```

Comment:
Not new to your PR, but I don't like seeing the general height declaration added _after_ the specific `.footer-nav-ipad & {` one. I would move this above just to be extra sure about the order of things.

Reply:
CSS will always prefer the 'most specific' selector, regardless of order. But totally happy to re-arrange if you think it makes it easier to read 👌

Reply:
Thanks!